src/modules/sensors/rc_update.cpp

```diff
@@ -350,9 +350,8 @@ RCUpdate::rc_poll(const ParameterHandles &parameter_handles)
 			manual.z = math::constrain(_filter_throttle.apply(manual.z), 0.f, 1.f);
 
 			if (_parameters.rc_map_flightmode > 0) {
-
 				/* the number of valid slots equals the index of the max marker minus one */
-				const int num_slots = manual_control_setpoint_s::MODE_SLOT_MAX;
+				const int num_slots = manual_control_setpoint_s::MODE_SLOT_NUM;
```

Comment:
Is the comment above still correct?

Reply:
No, it's not valid anymore 😬 
I fixed it by force push https://github.com/PX4/Firmware/compare/1464492d43143c8ff4a020caebda35a997e852cf..f698a05c8c5e306a4415da8adc2e5259108a5889

There are no special things to comment anymore.